stellarisdashboard/game_info.py

```diff
@@ -23,10 +27,15 @@ def load_name_mapping(self):
             with open(p, "rt", encoding="utf-8") as f:
                 for line in f:
                     try:
-                        key, val, *rest = line.strip().split('"')
-                        self.name_mapping[key.strip().rstrip(":0")] = val.strip()
-                    except Exception:
-                        pass
+                        re_match = self.loc_re.match(line)
+                        if re_match:
+                            self.name_mapping[ re_match.group('key') ] = re_match.group('value')
+                        else:
+                            if not line.startswith( ('#', "  ", " #", "  #", "   #", "\ufeffl_english:", "\n", " \n" ) ):
+                                # This error prints the offending line as numbers because not only did we encounter whitespace, we encountered the Zero Width No-Break Space (BOM)
+                                logger.debug("Unexpected unmatched localisation line found. Characters (as integers) follow: {}".format([ ord(y) for x,y in enumerate(line) ]))
```

Comment:
I guess this could just be
```suggestion
                                logger.debug(f"Unexpected unmatched localisation line found. Characters (as integers) follow: {[ord(x) for x in line]}")
```

Reply:
:heavy_check_mark: